### PR TITLE
change display value of ndvi from 0-255 to -1 to 1

### PR DESCRIPTION
**Pull request by Mridul97**
Changes are made to display value of ndvi in between -1 to 1 from 0 - 255.

### CODE REVIEW

**Comment by jywarren** [comment]
Hi, it looks like, since we recently merged in some new changes to the master branch, you need to rebase your work on top of the latest master. You can do that by following the process outlined here, but you may be able to skip step 2 if your master is still in sync with publiclab's master: https://publiclab.org/wiki/contributing-to-public-lab-software#Rewinding+the+master+branch

Thanks!

**Comment by Mridul97** [comment]
I did not make any changes to dist files and package.json but git is showing that some changes are made. Do I have to commit those changes as well?

**Comment by jywarren** [comment]
ah, if you run "grunt build" it will compile all your changes from /src/
into /dist/ accounting for the changes. Do the changes to /dist/ correspond
to those in /src/?

Let's see how they look once you push a commit up to this PR.

**Comment by Mridul97** [comment]
I have pushed the commits. Please take a look.

**Comment by Mridul97** [comment]
Actually I have not run the command "grunt build", I used "npm start", which I suppose further run "grunt serve" and there were changes made to the dist files. I don't exactly know that do changes in /dist/ corresponds to changes in /src/ as they are huge number of additions and deletions. I have rebased the branch. Please take a look. Thanks!

**Comment by jywarren** [comment]
OK, would you mind rebasing and pushing up the /dist/ changes too so I can see? Thanks!

**Comment by Mridul97** [comment]
I have pushed the /dist/ changes. Please have a look. Thanks!

**Comment by jywarren** [comment]
hmm, whoaaaaa can you try searching through the diff in the /dist/ file to
be sure it copied in your changes from src?

cool!

File: dist/image-sequencer.js [hunk 1]
```diff
@@ -47312,3374 +47312,3376 @@ arguments[4][38][0].apply(exports,arguments)
...
-},{}],152:[function(require,module,exports){
-module.exports = exports = function(pixels, blur) {
-    let kernel = kernelGenerator(blur, 1), oldpix = pixels;
-    kernel = flipKernel(kernel);
-
-    for (let i = 0; i < pixels.shape[0]; i++) {
-        for (let j = 0; j < pixels.shape[1]; j++) {
-            let neighboutPos = getNeighbouringPixelPositions([i, j]);
-            let acc = [0.0, 0.0, 0.0, 0.0];
-            for
```
Comment (jywarren) [inline dist/image-sequencer.js]: Here it is!!!

Comment (Mridul97) [reply dist/image-sequencer.js]: I found this!!
ss_5

Comment (Mridul97) [reply dist/image-sequencer.js]: This code is also getting displayed in additions.
ss_6

**Comment by Mridul97** [comment]
Yes, here is the screenshot!! 
ss_4

**Comment by jywarren** [comment]
Ah, great. One more change then -- can you truncate the NDVI value to 3 places, so `0.00`? That way it's more readable, too!

**Comment by Mridul97** [comment]
Cool! will do it. Thanks!

**Comment by Mridul97** [comment]
@jywarren Changes are not visible on browser, neither the ndvi changes nor the download button. They were working fine before. Even if I try to make any new change in index.html that is also not showing in the browser. The last thing I did in download button was rebasing it. Can't figure out why this is happening. Please help!

**Comment by jywarren** [comment]
hmm, do you maybe need to clear your browser cache? Could you try doing it
in an incognito window?

**Comment by Mridul97** [comment]
Yes, Its working fine in incognito window. So, there is no problem? Browser was just displaying stored page from the cache? I have truncated the NDVI values. Please take a look. Thanks!

**Comment by jywarren** [comment]
Great. Merging!

**Comment by jywarren** [comment]
Great work!

**Comment by jywarren** [comment]
@Mridul97 did you want to give this one a try? https://github.com/publiclab/image-sequencer/issues/281

**Comment by Mridul97** [comment]
Yes, I would like to work on this issue. Thanks!

**Pull request merged**